README.md

```diff
@@ -186,36 +198,40 @@ const {
   } = useScaffoldEventHistory({
   contractName: "YourContract",
   eventName: "GreetingChange",
-  fromBlock: //the block number to start reading events from,
+  // Specify the starting block number from which to read events.
+  fromBlock: 31231,
   blockData: true,
-  filters: //filters to be applied to the event (parameterName: value),
-  transactionData: //if set to true it will return the transaction data for each event (default: false),
-  receiptData: //if set to true it will return the receipt data for each event (default: false),
+  // Apply filters to the event based on parameter names and values (parameterName: value),
+  filters: parameterName: value,
```

Comment:
Maybe change this to something that is syntactically correct, e.g. 
```ts
// Apply filters to the event based on parameter names and values { [parameterName]: value },
filters: { premium: true }
```

Reply:
Thx a lot for the review @sverps! Just fixed it